ui/lib/core/addon/components/confirm-action.hbs

```diff
@@ -25,44 +25,13 @@
 {{/if}}
```

Comment:
since we're using confirm-modal in favor of confirm-action, do we still need this file?

Reply:
There's a comment in `confirm-modal` explaining the deprecation of this component. I can add something similar here to clear things up.

Reply:
No problem! Was just wondering if we wanted to remove it completely, but it seems like we're opting to keep it here for now since there might be some places that still use `ConfirmAction`?